### ISSUE TITLE
Doesn't work with ES6 imports?

### ISSUE THREAD

**Issue by liamcurry** [tivac/mithril-objectify#11]
I'm trying to use this as a browserify transform along with babelify but I'm running into some issues with ES6 imports.

This example works as expected:

```javascript
const m = require('mithril')
const tags = m('.hello', m('a.world', {href: '#'}, 'Hello!'))
console.log(tags)
```

But transforms aren't applied for this example:

```javascript
import m from 'mithril'
const tags = m('.hello', m('a.world', {href: '#'}, 'Hello!'))
console.log(tags)
```

**Comment by tivac** [comment]
I'll a look in a bit here, we aren't transpiling so not that surprising that it has issues. 😒

**Comment by tivac** [comment]
Fixed in `mithril-objectify@0.6.0`!
